code/iaas/labels/src/main/java/io/cattle/iaas/labels/service/LabelsService.java

```diff
@@ -0,0 +1,11 @@
+package io.cattle.iaas.labels.service;
+
+import io.cattle.platform.core.model.Label;
+
+public interface LabelsService {
+    Label getOrCreateLabel(Long accountId, String key, String value, String type);
+
+    void createContainerLabel(Long accountId, Long instanceId, String key, String value);
```

Comment:
should we change data type to primitive long? as you would expect them not to be null